src/pages/about-us/_DerivNumbers.tsx

```diff
@@ -0,0 +1,164 @@
+import React from 'react'
+import styled from 'styled-components'
+import { localize } from 'components/localization'
+import { SectionContainer, Flex, CssGrid } from 'components/containers'
+import { Header, Text } from 'components/elements'
+import device from 'themes/device'
+
+const StyledSection = styled(SectionContainer)`
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding: 200px 16px 120px;
+    background-color: var(--color-white);
+
+    @media ${device.tablet} {
+        padding: 40px 16px;
+    }
+`
+
+const StyledFlex = styled(Flex)`
+    margin: 0 2rem;
+    max-width: 1200px;
+    justify-content: space-around;
+
+    @media ${device.laptop} {
+        flex-direction: column;
+        align-items: center;
+    }
+`
+
+const TitleSection = styled(Flex)`
+    max-width: 46rem;
+    flex-direction: column;
+
+    @media ${device.laptop} {
+        margin-bottom: 8rem;
+        max-width: 90rem;
+        grid-column-gap: 4rem;
+    }
+
+    @media ${device.laptop} {
+        margin-bottom: 24px;
+    }
+`
+
+const NumberSection = styled(CssGrid)`
+    max-width: 588px;
+
+    @media ${device.tabletL} {
+        grid-column-gap: 4rem;
+    }
+
+    @media ${device.tabletS} {
+        grid-template-columns: minmax(auto, 484px);
+        grid-row-gap: 24px;
+        grid-template-rows: 1fr 1fr 1fr 1fr;
+        justify-content: start;
+    }
+`
+
+const TitleHeader = styled(Header)`
+    font-size: 48px;
+    line-height: 60px;
+
+    @media ${device.laptop} {
+        text-align: center;
+    }
+
+    @media ${device.tabletS} {
+        text-align: left;
+    }
+
+    @media ${device.mobileL} {
+        width: auto;
+        text-align: left;
+    }
+`
```

Comment:
Please use the correct `type` accordingly and pass the text align and width in as props.